include/os/freebsd/spl/sys/dkio.h

```diff
@@ -1,494 +1,39 @@
 /*
- * CDDL HEADER START
+ *  Copyright (C) 2007-2010 Lawrence Livermore National Security, LLC.
+ *  Copyright (C) 2007 The Regents of the University of California.
+ *  Produced at Lawrence Livermore National Laboratory (cf, DISCLAIMER).
+ *  Written by Brian Behlendorf <behlendorf1@llnl.gov>.
+ *  UCRL-CODE-235197
  *
- * The contents of this file are subject to the terms of the
- * Common Development and Distribution License (the "License").
- * You may not use this file except in compliance with the License.
+ *  This file is part of the SPL, Solaris Porting Layer.
```

Comment:
FreeBSD likes to keep its SPL under the CDDL while Linux likes to keep it under the GPL. I think we should just ask @behlendorf to sign off on relicensing the file to CDDL to be consistent. Then we can deal with the dual license when the two are eventually unified. If we are using the Linux version here, we probably could move this header into a common location, such as the currently non-existent `include/spl/sys/dkio.h`. If we do that, we probably should dual-license the file CDDL/GPL just to maintain the "license purity" of the two SPLs, even though headers should not to be subject to copyright (at least, I was advised by legal counsel to continue thinking that the last time I asked about it circa ~2016), so what "license" is used here is an extremely trivial thing.

Reply:
Oops, I hadn't noticed the license change. I rechecked that diff and it's easy to just trim the CDDL version (it turns out that the Linux version is actually bloated by 50%) so I'll push an update with that momentarily.